backend.native/tests/interop/objc/allocException.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2010-2019 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license
+ * that can be found in the LICENSE file.
+ */
+
+import platform.Foundation.*
+import kotlin.test.*
+
+import kotlinx.cinterop.*
+import platform.posix.*
+import kotlin.system.exitProcess
+
+fun sigabrt_handler(signum: Int) : Unit {
+    println("Exception handled successfully, signum = $signum")
+    exitProcess(0)
+}
+
+class A : NSJSONSerialization()
```

Comment:
Subclassing is likely not required here, instantiating `NSJSONSerialization()` should be enough to reproduce the original issue.